database/psql.js

```diff
@@ -0,0 +1,247 @@
+const inputs = require('../server/inputs.js');
+const {getTrialEndDate} = require('../helpers/getTrialEndDate.js');
+const Sequelize = require('sequelize');
+const sequelize = new Sequelize({
+  database: 'test',
+  dialect: 'postgres',
+  username: 'abscura',
+  password: null,
+  operatorsAliases: Sequelize.Op,
+});
+
+sequelize
+  .authenticate()
+  .then(() => {
+    console.log('\x1b[32m----- SUCCESSFUL DATABASE CONNECTION -----\x1b[0m');
+  })
+  .catch(err => {
+    console.error('\x1b[31m----- FAILED DATABASE CONNECTION -----\x1b[0m');
+    console.error(err);
+  });
+
+
+/* -------------------- TABLE DEFINITION -------------------- */
+
+
+const Users = sequelize.define('users',
+  {
+    id: {
+      type: Sequelize.INTEGER,
+      primaryKey: true,
+      unique: true,
+      autoIncrement: true
+    },
+    first_name: { type: Sequelize.STRING },
+    last_name: { type: Sequelize.STRING },
+    email: { type: Sequelize.STRING },
+    contact_number: { type: Sequelize.STRING }
+  },
+  { timestamps: false }
+);
+
+const Addresses = sequelize.define('addresses',
+  {
+    id: {
+      type: Sequelize.INTEGER,
+      primaryKey: true,
+      unique: true,
+      autoIncrement: true
+    },
+    user_id: { type: Sequelize.STRING },
+    full_name: { type: Sequelize.STRING },
+    line_1: { type: Sequelize.STRING },
+    line_2: { type: Sequelize.STRING },
+    city: { type: Sequelize.STRING },
+    state: { type: Sequelize.STRING },
+    zipcode: { type: Sequelize.STRING },
+    country: { type: Sequelize.STRING }
+  },
+  { timestamps: false }
+);
+
+const PrimeMemberships = sequelize.define('primememberships',
+  {
+    id: {
+      type: Sequelize.INTEGER,
+      primaryKey: true,
+      unique: true,
+      autoIncrement: true
+    },
+    user_id: { type: Sequelize.STRING },
+    prime_status: { type: Sequelize.STRING },
+    total_spend_trial_signup: { type: Sequelize.INTEGER },
+    trial_signup_date: { type: Sequelize.DATEONLY },
+    trial_end_date: { type: Sequelize.DATEONLY },
+    prime_cancel_date: { type: Sequelize.DATEONLY }
+  },
+  { timestamps: false }
+);
+
+const PrimeRetentionAnalysis = sequelize.define('primeretentionanalysis',
+  {
+    id: {
+      type: Sequelize.INTEGER,
+      primaryKey: true,
+      unique: true,
+      autoIncrement: true
+    },
+    user_id: { type: Sequelize.STRING },
```

Comment:
Same comment as above -- this might need to be an integer if it's going to be used to join tables where user_id is an integer.

Reply:
Fixed in upcoming PR!